RobotCode/snobot2018/src/org/snobot/PortMappings2018.java

```diff
@@ -1,19 +1,22 @@
-package org.snobot;

```

Comment:
Line endings are bad.  Maybe pulling master on top will help.  This isn't the first time this file (and the one below) have had this happen.  We should fix it before it gets more out of control